src/Components/SingleArticle.jsx

```diff
@@ -1,23 +1,24 @@
-import { useState, useEffect } from 'react'
-import { useParams, Link, useNavigate, useSearchParams } from 'react-router-dom'
+import { useState, useEffect } from 'react';
+import { useParams, Link, useNavigate, useSearchParams } from 'react-router-dom';
 import { getArticleById, getCommentsByArticleId } from "../api";
-import { Share, MessageCircle, ArrowLeft, ArrowBigUp, ArrowBigDown } from 'lucide-react';
+import { Share, MessageCircle, ArrowLeft } from 'lucide-react';
 import { formatDate } from '../../utils/formatting';
 import CommentCard from './CommentCard';
+import VoteButtons from './VoteButtons'; // Import the VoteButtons component
 
 const SingleArticle = () => {
-  const navigate = useNavigate();
   const { article_id } = useParams();
+  const navigate = useNavigate();
   const [searchParams] = useSearchParams();
-  const showCommentsParam = searchParams.get('showComments');
+  const showCommentsInitially = searchParams.get('showComments') === 'true';
```

Comment:
Do you have a searchPAram called showComments? I don't think you do, so this will always equal false... so why not just set your state on line 20 to false? It isn't clear why you are using searchParams.